### DISCUSSION TITLE
Bulk Reboot Devices

### DISCUSSION THREAD

**Discussion by vaijanaths007** [kctrey/wxcadm#4] [General]
Hi Trey,
Does this program include an option to reset devices in bulk?

Thanks,
Vaijanath S

**Comment by kctrey** [comment]
No. At the moment, the Webex Devices API doesn't support it, so I don't have a method to do it. There would be some "back door" ways to do it, but with the Devices API (supposedly) coming soon, I haven't developed it.

If you want to do it yourself, you can open the Dev Tools in the browser while doing a device apply/reboot and you'll see the exact API call that Control Hub makes, so you could do that now until the official API comes out. The trick is that it needs a Control Hub auth token; you can't use an Integration token to do it because the scope isn't available for Integrations.
